src/Redux/categories/categories.js

```diff
@@ -0,0 +1,20 @@
+const CHECK_STATUS = 'checkStatus';
```

Comment:
- [ ] According to the requirements, kindly ensure that the values of these action types follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux). It should look something like this:
```const CHECK_STATUS = bookstore/categories/CHECK_STATUS```